scripts/mysqld_safe.sh

```diff
@@ -21,9 +21,11 @@ mysqld_ld_library_path=
 flush_caches=0
 numa_interleave=0
 wsrep_on=0
+simulate=0
```

Comment:
Yes systemd can run user instances - https://wiki.archlinux.org/index.php/Systemd/User. KDE Akonadi is the only regular user demand I can think of however I'm not 100% sure what that definition means to you. The compilation of socket activation doesn't prevent launching as mysqld like an init script, using mysqld_safe, or mysqld from the command_line like before this patch.